app/controllers/api/v1/users_controller.rb

```diff
@@ -139,6 +142,13 @@ def user_attributes
         user_params.except(:invitation)
```

Comment:
Je pense qu'on pourrait directement faire la conversion dans cette méthode (et ajouter les `creation_origin_attibutes` aussi d'ailleurs) et on aurait qq chose comme ça
```suggestion
     def user_attributes
        user_params.except(:invitation).merge(creation_origin_attributes).tap { |attrs| convert_tags_to_add_to_tag_users_attributes(attrs) }
     end
     
     def convert_tags_to_add_to_tag_users_attributes(attrs)
        return if attrs[:tags_to_add].blank?
        
        ...
      end
```

et on aurait pas besoin de la méthode `convert_tags_for_upsert_service!`